code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-eng"
+	dispensedKit = /obj/item/storage/box/staffkit/eng
+
+/datum/recruitment_role/mechanic
+	name = "Mechanics Assistant"
+	accessParent = "Mechanic"
+	canAuthorize = list("Mechanic","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mech"
+	dispensedKit = /obj/item/storage/box/staffkit/mech
+
+/datum/recruitment_role/mining
+	name = "Mining Assistant"
+	accessParent = "Miner"
+	canAuthorize = list("Miner","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mine"
+	dispensedKit = /obj/item/storage/box/staffkit/miner
+
+/datum/recruitment_role/sci
+	name = "Research Assistant"
+	accessParent = "Scientist"
+	canAuthorize = list("Scientist","Research Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-sci"
+	dispensedKit = /obj/item/storage/box/staffkit/res
+
+/datum/recruitment_role/chef
+	name = "Culinary Assistant"
+	accessParent = "Chef"
+	canAuthorize = list("Chef","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-chef"
+	dispensedKit = /obj/item/storage/box/staffkit/chef
+
+/datum/recruitment_role/bar
+	name = "Bar Assistant"
+	accessParent = "Bartender"
+	canAuthorize = list("Bartender","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-bar"
+	dispensedKit = /obj/item/storage/box/staffkit/bar
+
+/datum/recruitment_role/botany
+	name = "Botanical Assistant"
+	accessParent = "Botanist"
+	canAuthorize = list("Botanist","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-hyd"
+	dispensedKit = /obj/item/storage/box/staffkit/botany
+
+
+//equipment kits
+
+/obj/item/storage/box/staffkit/med
+	name = "medical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/reagent_containers/emergency_injector/epinephrine,
+		/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/bandage)
+
+/obj/item/storage/box/staffkit/robo
+	name = "robotics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
+
+/obj/item/storage/box/staffkit/eng
+	name = "engineering assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/engineer,
+		/obj/item/clothing/head/helmet/hardhat,
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
```

Comment:
With all the discussion regarding multitool availability, this should be covered by map.
Add upgraded atmos analyzer perhaps?
Maybe engineers should start with hardhats... 🤔

Reply:
Engineers and Mechanics always have multitools they are essential for the job

Reply:
Multitool scarcity is not supposed to be "nobody should get their hands on them" - more of "if you want to get your hands on them, you need to ask someone in a pertinent department, or QM". As the tokens have to be authorized by appropriate departmental members, this seems to fit pretty nicely into that design space. Upgraded atmos analyzer's a good call.

Reply:
From a QoL standpoint I agree it would be easier to just give them the things that the job starts with.  I think there are enough multi-tools to go around, you just have to talk to someone to get one.  I'd rather see the staff assistant request token authorization and have to come back to the department to gear up for crime instead of being able to request authorization and then never be seen again.

Effort required to tide and antag as staff assistant seems GREATLY reduced by being able to say "Engineer/Mech plz" to HoP at round start.

Reply:
Still on the fence, but I think I'm sold. Will probably replace the engineer multitool with the upgraded atmos analyzer and drop the mechanic multitool.

Reply:
I really would not advise this engineers and mechanics often do not have any multi-tools left because they're taken. This is standard equipment they should both have to do their jobs. I don't know what AzRun is talking about, both jobs START with multitools

Reply:
Re-reconsidered? I'll probably have to ask around on this topic.